File: Programmers/최소직사각형.py
def solution(sizes):
    w_check = 0
    h_check = 0
    for i in range(len(sizes)):
        if sizes[i][0] > sizes[i][1]:
            w_check = max(w_check,sizes[i][0])
            h_check = max(h_check,sizes[i][1])
        else:
            w_check = max(w_check,sizes[i][1])
            h_check = max(h_check,sizes[i][0])
    return w_check*h_check

File: Programmers/최소직사각형.swift
import Foundation

func solution(_ sizes:[[Int]]) -> Int {
    var maxWidth = 0, maxHeight = 0
    
    for size in sizes {
        if size[0] > size[1] {
            maxWidth = max(maxWidth, size[0])
            maxHeight = max(maxHeight, size[1])
        } else {
            maxWidth = max(maxWidth, size[1])
            maxHeight = max(maxHeight, size[0])
        }
    }
    
    return maxWidth * maxHeight
}
